pkg/core/errors.go

```diff
@@ -24,4 +24,5 @@ var (
 	ErrCanceledByUser   = errors.New("canceled by user")
 	ErrInvalidPID       = errors.New("PID does not exist")
 	ErrInvalidContainer = errors.New("container does not exist")
+	ErrBatteryNotFound  = errors.New("could not read from /sys/class/power_supply/BAT0")
```

Comment:
nit: can this be worded a little better? Something like "Battery not found" or "battery information could not be read" perhaps? Assuming this will be shown to the user.

Reply:
This error won't be visible to the user though, if this error occurs, we simply allow the battery gauge to say "Battery Not Found", and the rest of the UI still renders. This is because, in lots of instances, this information might not exist and thus grofer will exit unnecessarily.

Reply:
Ah right, makes sense. Nvm then